src/libraries/Directory.Build.props

```diff
@@ -119,16 +97,16 @@
   <PropertyGroup>
     <!-- By default make all libraries to be AnyCPU but individual projects can override it if they need to -->
     <Platform>AnyCPU</Platform>
-    <OutputType>Library</OutputType>
```

Comment:
Just curious - where did this go?

Reply:
this is the default, no need to be explicit here.